src/v2/view-builder/views/AutoRedirectView.js

```diff
@@ -1,30 +1,11 @@
-import { _, loc } from 'okta';
+import { loc } from 'okta';
 import { BaseForm, BaseView } from '../internals';
 
 const Body = BaseForm.extend({
   title() {
-    let titleString = loc('oie.success.text.signingIn', 'login');
-    // For more info on the API response available in appState, see IdxResponseBuilder.java
-    const app = this.options.appState.get('app');
-    const user = this.options.appState.get('user');
-
-    if (_.isEmpty(app)) {
-      return titleString;
-    }
-
-    const { label: appInstanceName, name: appDisplayName } = app;
-    const { identifier: userEmail } = user || {};
-
-    const appName = appInstanceName ? appInstanceName : appDisplayName;
-
-    if (appName && userEmail) {
-      titleString = loc('oie.success.text.signingIn.with.appName.and.identifier', 'login', [appName, userEmail]);
-    } else if (appName) {
-      titleString = loc('oie.success.text.signingIn.with.appName', 'login', [appName]);
-    }
-
-    return titleString;
+    return loc('oie.success.text.signingIn.with.fastpass', 'login');
```

Comment:
we seem to be removing some useful info here. whats the reasoning ?

Reply:
@magizh-okta it's from the mocks here - https://www.figma.com/file/wra4gnCX3Xj7HnwCFNXQ0A/Okta-FastPass-(WIP)?node-id=464%3A22851

i'll let @panchamyadav-okta answer this